--- a/controllers/config/updateCSS.js	2026-04-24 17:37:52.000000000 +0200
+++ b/controllers/config/updateCSS.js	2026-04-25 14:10:47.186296208 +0200
@@ -11,10 +11,10 @@
   file.write(req.body.styles, false);
 
   // Copy file to docker volume
-  fs.copyFileSync(
-    join(__dirname, '../../public/flame.css'),
-    join(__dirname, '../../data/flame.css')
-  );
+  // fs.copyFileSync(
+  //   join(__dirname, '../../public/flame.css'),
+  //   join(__dirname, '../../data/flame.css')
+  // );
 
   res.status(200).json({
     success: true,
--- a/db/migrations/01_new-config.js	2026-04-24 17:37:52.000000000 +0200
+++ b/db/migrations/01_new-config.js	2026-04-25 14:19:06.436753700 +0200
@@ -1,8 +1,10 @@
+const fs = require('fs');
 const { readFile, writeFile, copyFile } = require('fs/promises');
 const Config = require('../../models/Config');
 
 const up = async (query) => {
   await copyFile('utils/init/initialConfig.json', 'data/config.json');
+  fs.chmodSync('data/config.json', 0o644);
 
   const initConfigFile = await readFile('data/config.json', 'utf-8');
   const parsedNewConfig = JSON.parse(initConfigFile);
@@ -25,6 +27,7 @@
 
   const newConfig = JSON.stringify(parsedNewConfig);
   await writeFile('data/config.json', newConfig);
+  fs.chmodSync('data/config.json', 0o644);
 
   await query.dropTable('config');
 };
--- a/middleware/multer.js	2026-04-24 17:37:52.000000000 +0200
+++ b/middleware/multer.js	2026-04-25 14:14:50.995732796 +0200
@@ -1,8 +1,8 @@
 const fs = require('fs');
 const multer = require('multer');
 
-if (!fs.existsSync('data/uploads')) {
-  fs.mkdirSync('data/uploads', { recursive: true });
+if (!fs.existsSync('./data/uploads')) {
+  fs.mkdirSync('./data/uploads', { recursive: true });
 }
 
 const storage = multer.diskStorage({
--- a/utils/init/createFile.js	2026-04-24 17:37:52.000000000 +0200
+++ b/utils/init/createFile.js	2026-04-25 14:16:32.626690376 +0200
@@ -13,6 +13,7 @@
   // Check if file exists
   if (fs.existsSync(srcPath)) {
     fs.copyFileSync(srcPath, destPath);
+    fs.chmodSync(destPath, 0o644);
 
     if (process.env.NODE_ENV == 'development') {
       logger.log(msg.found);
@@ -23,6 +24,7 @@
 
   // Create file if not
   fs.writeFileSync(destPath, isJSON ? JSON.stringify(template) : template);
+  fs.chmodSync(destPath, 0o644);
 
   if (process.env.NODE_ENV == 'development') {
     logger.log(msg.created);
--- a/utils/init/index.js	2026-04-24 17:37:52.000000000 +0200
+++ b/utils/init/index.js	2026-04-25 14:16:53.474497269 +0200
@@ -1,11 +1,11 @@
 const initConfig = require('./initConfig');
 const initFiles = require('./initFiles');
-const initDockerSecrets = require('./initDockerSecrets');
+// const initDockerSecrets = require('./initDockerSecrets');
 const initSecret = require('./initSecret');
 const normalizeTheme = require('./normalizeTheme');
 
 const initApp = async () => {
-  initDockerSecrets();
+  // initDockerSecrets();
   initSecret();
   await initFiles();
   await initConfig();
--- a/utils/init/initConfig.js	2026-04-24 17:37:52.000000000 +0200
+++ b/utils/init/initConfig.js	2026-04-25 14:17:44.130523707 +0200
@@ -1,3 +1,4 @@
+const fs = require('fs');
 const { copyFile, readFile, writeFile } = require('fs/promises');
 const checkFileExists = require('../checkFileExists');
 const initialConfig = require('./initialConfig.json');
@@ -7,6 +8,7 @@
 
   if (!configExists) {
     await copyFile('utils/init/initialConfig.json', 'data/config.json');
+    fs.chmodSync('data/config.json', 0o644);
   }
 
   const existingConfig = await readFile('data/config.json', 'utf-8');
@@ -20,6 +22,7 @@
   }
 
   await writeFile('data/config.json', JSON.stringify(parsedConfig));
+  fs.chmodSync('data/config.json', 0o644);
 };
 
 module.exports = initConfig;
--- a/utils/init/initialFiles.json	2026-04-24 17:37:52.000000000 +0200
+++ b/utils/init/initialFiles.json	2026-04-25 14:18:25.521913694 +0200
@@ -1,19 +1,6 @@
 {
   "files": [
     {
-      "name": "flame.css",
-      "msg": {
-        "created": "Created empty CSS file",
-        "found": "Custom CSS file found"
-      },
-      "paths": {
-        "src": "../../data",
-        "dest": "../../public"
-      },
-      "template": "",
-      "isJSON": false
-    },
-    {
       "name": "customQueries.json",
       "msg": {
         "created": "Created empty queries file",
